R-packages/covidcast/R/plot.R

```diff
@@ -713,60 +697,60 @@ plot_line = function(x, range = NULL, title = NULL, params = list()) {
 }
 
 
-get_pr_centroid = function(map_df){
-  pr_df = map_df %>% dplyr::filter(.$is_pr) %>% sf::st_transform(., 102003)
-  pr_centroid = sf::st_centroid(sf::st_geometry(pr_df))
-  return(pr_centroid)
-}
 
+# Use the following CRS values
```

Comment:
Are these derived from some external source or did you just tinker around to find these values?  Would be good to give more context

Reply:
Context given.